src/test/load-action-certificate.spec.js

```diff
@@ -1,6 +1,6 @@
 import { Selector } from "testcafe";
 
-fixture("Load action from plain certificate").page`http://localhost:3000`;
+fixture.skip("Load action from plain certificate").page`http://localhost:3000`;
```

Comment:
of course you fix this in the next MR :D

Reply:
ctually I prefer the opposite, fix the test first then merge this MR :)

Reply:
ok fixed by replacing a gist sample instead of myjson